### PR TITLE
Initial templates

### PR DESCRIPTION
**Pull request by flawedworld**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+https://github.com/GrapheneOS/Apps
+
+https://github.com/GrapheneOS/Camera
+
+https://github.com/GrapheneOS/PDFViewer
+
+## Guidance
+
+Not all translations are simple, and some must be able to convery technical intricacy towards users who lack context.
```
Comment (e-zk) [inline README.md]: typo: `convery`, should be `convey`.

Comment (flawedworld) [reply README.md]: thnx

Comment (e-zk) [reply README.md]: ✌

**Review by PatrykMis — APPROVED** [review]

**Comment by flawedworld** [comment]
On hold for now, this will be revisited.

**Pull request closed without merging**